ui/ui-components/pages/model/[modelId]/source/[sourceId]/multipleProperties.tsx

```diff
@@ -374,7 +367,7 @@ const Page: NextPageWithInferredProps<typeof getServerSideProps> = ({
           <Table>
             <thead>
               <tr>
-                <th>Source Options</th>
+                <th>Source Option</th>
```

Comment:
That's a good point!  With some sources it might not be a column (mongo), but we've already made the choice to use that term... it will be more clear for most users.